woof-code/packages-templates/wireplumber_FIXUPHACK

```diff
@@ -1,6 +1,6 @@
 rm -f `find . -name libwireplumber-module-logind.so` 2>/dev/null
 
-if [ -f /etc/wireplumber/wireplumber.conf.d/bluez-nologind.conf ]; then
+if [ ! -f /etc/wireplumber/wireplumber.conf.d/bluez-nologind.conf ]; then
```

Comment:
Are you sure that `/etc/wireplumber/wireplumber.conf.d/` always exists?

Also, as I said in my previous review, and also as @dimkr said, this change will run on the build system rather than the targeted build since FIXUPHACKS are not chrooted.

You can try running a test build to see if your PR succeeds in doing its task or not.

Reply:
Added folder check. If not exists create new onw